examples/apps/echo/client/echo_client.py

```diff
@@ -124,7 +131,13 @@ def ParseCommandLine(args):
 
     requester_signature = options.requester_signature
     input_data_hash = options.data_hash
+    # worker id
     worker_id = options.worker_id
+    worker_id_hex = options.worker_id_hex
+
+    worker_id = worker_id_hex if not worker_id \
+        else hex_utils.get_worker_id_from_name(worker_id)
+
```

Comment:
It would be good if we validate the worker id passed in the command line is correct or not?

Reply:
Worker_id can be any name. We don't have anything to compare against.